### DISCUSSION TITLE
Fiz a instalação normal do vscode porém deu um problema no shell

### DISCUSSION THREAD

**Discussion by AmandaDiasF** [balta-io/forum#715] [Tira Dúvidas] [answered]
Após instalar mesmo reeniciando, ele n aparece o shell command, Curso C# introdução, instalei o dotnet 8, ja tinha o 9 tbm por conta de ter o Visual Studio, poderia ser algum conflito? do vscode com o visual Studio?

![erro](https://github.com/user-attachments/assets/d63c7c5d-0c06-4220-943c-0593db1f7557)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Agdakelda, como vai?

Normalmente não conflita ter o VSCode e VS juntos não, inclusive tenho no Windows e é super de boa.

O que deveria aparecer no command windows que não está sendo exibido? Não entendi muito bem o que deu errado!

**Reply by AmandaDiasF** [reply]
então no video após a instalação do shell e do vscode ao executar o comando para achar o shell command ele não fica listado como no do video consigo continuar sem ele aparecer por la ? fica como no print eu pesquiso e ele n aparece

**Reply by andrebaltieri** [reply]
@Agdakelda você diz o comando para instalar o VSCode no Path do Windows (Variáveis de ambiente)?
- Se sim, procura por "path" na mesma caixa que está pesquisando o Shell Command... ele vai aparecer algo como "Add to Path"... Qualquer coisa manda um print aqui.

Dá pra continuar sem isso sim, a única coisa que esse comando faz é adicionar o `vscode.exe` as variáveis de ambiente do Windows. Com isto, sempre que estiver no terminal, poderá usar o comando `code` igual faço nos vídeos!
